src/packages/calendaritem/calendaritem.taro.tsx

```
@@ -560,159 +529,126 @@ export const CalendarItem = React.forwardRef<
     day: CalendarDay,
     month: CalendarMonthInfo,
-    isFirst?: boolean
+    isFirst: boolean = true
   ) => {
-    if (getClasses(day, month) === `${dayPrefix}-disabled`) {
-      return
-    }
-
+    if (isDisable(day, month) || disableDate(day)) return
     const days = [...month.curData]
     const [y, m] = month.curData
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)